server/util/db/db.go

```diff
@@ -110,24 +114,21 @@ func (dbh *DBHandle) ReadRow(out interface{}, where ...interface{}) error {
 	return err
 }
 
-func maybeRunMigrations(dialect string, gdb *gorm.DB) error {
-	if *autoMigrateDB {
-		postAutoMigrateFuncs, err := tables.PreAutoMigrate(gdb)
-		if err != nil {
-			return err
-		}
-		gdb.AutoMigrate(tables.GetAllTables()...)
-		if postAutoMigrateFuncs != nil {
-			for _, f := range postAutoMigrateFuncs {
-				if err := f(); err != nil {
-					return err
-				}
-			}
-		}
-		if err := tables.PostAutoMigrate(gdb); err != nil {
+func runMigrations(dialect string, gdb *gorm.DB) error {
+	log.Info("Auto-migrating DB")
+	postAutoMigrateFuncs, err := tables.PreAutoMigrate(gdb)
+	if err != nil {
+		return err
+	}
+	gdb.AutoMigrate(tables.GetAllTables()...)
```

Comment:
Since we're updating this code, let's add a check for the returned error from GORM.

Reply:
Good eye -- fixed